Libraries/Boards/MAX32690/EvKit_V1/Source/board.c

```diff
@@ -101,6 +101,18 @@ __weak void GPIOWAKE_IRQHandler(void)
     MXC_GPIO_Handler(MXC_GPIO_GET_IDX(MXC_GPIO4));
 }
 
+// Default handler for generic GPIO interrupts on port 2 (used by GPIO example)
+__weak void GPIO2_IRQHandler(void)
+{
+    MXC_GPIO_Handler(MXC_GPIO_GET_IDX(MXC_GPIO2));
+}
+
+// Default handler for generic GPIO interrupts on port 4 (used by pushbutton)
+__weak void GPIO4_IRQHandler(void)
```

Comment:
I believe GPIOWAKE is the interrupt vector for GPIO4.

Reply:
Thanks, removed in https://github.com/Analog-Devices-MSDK/msdk/pull/712/commits/e926cf21f0a52a935583087f438bdf8e0cf3237e